packages/tools/addon/components/cs-field-overlays.js

```diff
@@ -14,10 +15,20 @@ export default Component.extend({
   cardstackRouting: service(),
   router: service(),
 
+  selectedOverlayState: null,
+  isOverlayOpen: equal('selectedOverlayState', 'open'),
+  isOverlayPassthrough: equal('selectedOverlayState', 'passthrough'),
+
   actions: {
     openField(which) {
+      if (!which) {
+        this.set('selectedOverlayState', 'closed');
+      } else {
+        this.set('selectedOverlayState', this.isOverlayOpen ? 'passthrough' : 'open');
```

Comment:
I don't think we should keep this state (`selectedOverlayState`) in this component. It's redundant with state elsewhere, which means it's at high risk for bugs. I also don't think it's supposed to be shared by every overlay, which is the current effect.

Instead, simplify to:

```
class=(concat "cs-field-overlay "
              (if hasInlineEditor "with-inline-editor ")
              (if (eq mark.id tools.openedFieldId) "opened"))
```

Then you can do something like:

```css
.cs-field-overlay.with-inline-editor.opened {
  pointer-events: none;
}
```